mkosi/kmod.py

```diff
@@ -8,12 +8,18 @@
 import subprocess
 from collections.abc import Iterable, Iterator, Reversible
 from pathlib import Path
+from typing import TypedDict
 
 from mkosi.context import Context
 from mkosi.log import complete_step, log_step
 from mkosi.run import chroot_cmd, run
 from mkosi.sandbox import chase
-from mkosi.util import chdir, parents_below
+from mkosi.util import chdir, flatten, parents_below
+
+
+class DepsDict(TypedDict):
```

Comment:
Also, let's make this a dataclass, since we generally use dataclasses for stuff like this.